### PR TITLE
[SDK] Add start and limit to get events by handle api

### PR DESCRIPTION
**Pull request by areshand**
## Motivation
so that we can set the start and limit when reading events

(You must have submitted a [signed CLA](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md#contributor-license-agreement) that includes your GitHub handle prior to us accepting and landing your work. Write your answer here.)

## Test Plan

(Share your test plan here. If you changed code, please provide us with clear instructions for verifying that your changes work.)
added new unit test

### CODE REVIEW

**Comment by areshand** [comment]
@CapCap  could you take a look?  do we also need to publish the change to npm after landing it?

**Comment by CapCap** [comment]
@areshand would need to bump minor version, and publish after landing, yeah!

**Review by CapCap — APPROVED** [review]
Small nit around limits, and please bump the NPM minor version before landing :-)

File: api/doc/openapi.yaml [hunk 1]
```diff
@@ -488,6 +490,22 @@ components:
...
+      in: query
+      required: false
+      description: The start index of events emitted from the account
+      example: 0
+      schema:
+        type: integer
+    LimitEvent:
+      name: limit
+      in: query
+      required: false
```
[the remark this answers is not in the document]
Comment (areshand) [reply api/doc/openapi.yaml:504]: used a random number from head.  updated to 25

**Comment by areshand** [comment]
> @areshand would need to bump minor version, and publish after landing, yeah!

updated to 0.0.16. For publishing the package, do I need to have obtain some permission before being able to publish this npm package?

**Comment by CapCap** [comment]
> > @areshand would need to bump minor version, and publish after landing, yeah!
> 
> updated to 0.0.16. For publishing the package, do I need to have obtain some permission before being able to publish this npm package?

I think right now I’m the only one with access to publish… setting it up in CI is a todo :-) I can manually publish it out after landing

File: api/doc/openapi.yaml [hunk 2]
```diff
@@ -488,6 +490,22 @@ components:
       example: 25
       schema:
         type: integer
+    StartEvent:
+      name: start
+      in: query
+      required: false
+      description: The start index of events emitted from the account
```
Comment (msmouse) [inline api/doc/openapi.yaml:497]: The start sequence number in the EVENT STREAM, defaulting to the latest event. The events are returned in the reverse order of sequence numbers.

File: api/doc/openapi.yaml [hunk 3]
```diff
@@ -442,6 +442,8 @@ paths:
           schema:
             type: string
           example: "sent_events"
+        - $ref: '#/components/parameters/StartEvent'
+        - $ref: '#/components/parameters/LimitEvent'
```
Comment (msmouse) [inline api/doc/openapi.yaml:446]: `EventsStart` and `EventsLimit` sounds slightly more natural to me.. I'm not sure though..

File: api/doc/openapi.yaml (hunk 2 above)
Comment (areshand) [reply api/doc/openapi.yaml]: updated the description and remove the default 0 for start

File: api/doc/openapi.yaml (hunk 3 above)
Comment (areshand) [reply api/doc/openapi.yaml]: updated the names

**Pull request closed without merging**